corehq/apps/export/esaccessors.py

```diff
@@ -26,6 +27,12 @@ def get_case_export_base_query(domain, case_type):
             .sort("opened_on"))
 
 
+def get_sms_export_base_query(domain):
+    return (SMSES()
+            .domain(domain)
+            .sort("date"))
```

Comment:
i forgot to mention, we should also exclude any messages that are outbound and had errors. we can do that by excluding those messages with `direction` equal to [OUTGOING](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/models.py) and `processed` False